src/config/ConfigManager.cpp

```diff
@@ -4,8 +4,14 @@
 CConfigManager::CConfigManager() {
     // init the entire thing
 
-    const char* const ENVHOME = getenv("HOME");
-    const std::string CONFIGPATH = ENVHOME + (std::string) "/.config/hypr/hyprpaper.conf";
+    std::string CONFIGPATH;
```

Comment:
CAPS is for `const` vars

Reply:
Same goes for [`CONFIGPATH`](https://github.com/hyprwm/Hyprland/blob/main/src/config/ConfigManager.cpp?plain=1#L17) in hyprland codebase which is also all caps, while not being a constant.

Reply:
it's meant to be a constant but cannot be due to some long if assignments.

Reply:
It was like this in few other spots actually ([`CAIROSURFACE`](https://github.com/hyprwm/hyprpaper/blob/main/src/render/WallpaperTarget.cpp?plain=1#L12) for another example), so application is pretty inconsistent code style-wise. Changed to `configPath` now.